ldap/src/main/java/org/springframework/security/ldap/DefaultSpringSecurityContextSource.java

```diff
@@ -16,6 +16,7 @@
 
 package org.springframework.security.ldap;
 
+import java.net.URI;
```

Comment:
Please update the copyright years for all files in the PR. For example, this one would change from `2002-2016` to `2002-2021`.